proposals/i18n-getOSLanguage.md

```diff
@@ -0,0 +1,78 @@
+# Proposal: i18n.getOSLanguage
+
+**Summary**
+
+Allow to get the language of your operating system as an [BCP47 language tag](https://www.w3.org/International/core/langtags/rfc3066bis.html).
```

Comment:
This is a really ancient link (thank you for discovering it so that we/i18n can fix it). Consider linking to the real deal:

```suggestion
Allows getting the language/locale of your operating system as a [BCP47 language tag](https://www.rfc-editor.org/bcp/bcp47.html)
```

I18N discussed this issue in our call of 2024-04-18. I will add comments to #252.

Reply:
Thanks for the update @aphillips ! Great to hear it was discussed. looking forward to the comments.

Reply:
@aphillips Updated the PR to address some of the feedback raised in #252. Namely splitting the method into two separate ones with better naming. See https://github.com/w3c/webextensions/pull/569#issuecomment-2126669092

Seems all vendors have no objection to the changes and this PR. Is there anything you would like to add here?